src/rustllvm/RustWrapper.cpp

```diff
@@ -1000,6 +1000,20 @@ extern "C" bool LLVMRustAddYkBlockLabel(LLVMBuilderRef Builder,
     return true;
 }
 
+extern "C" unsigned int LLVMRustInstructionIndex(LLVMValueRef Value) {
```

Comment:
This method returns the wrong result if the instruction is not found.

We should either add a comment that states that the instruction must be present, or we can make this crash or return an error if it isn't found.

Reply:
I don't see how that can happen. I don't think there can be an instruction without a parent basic block. And if there is a parent basic block, then we will find that instruction inside it.

Reply:
You are right. My bad.